rna-transcription.js

```diff
@@ -0,0 +1,13 @@
+var rnaComplementMapping = { C: 'G', G: 'C', A: 'U', T: 'A' };
+
+function rnaComplement(dnaNucleotide) {
+  return rnaComplementMapping[dnaNucleotide];
+}
+
+function convertDnaNucleotidesToRna(dnaNucleotides) {
+  return dnaNucleotides.replace(/./g, rnaComplement);
```

Comment:
Maybe try to write a test to prevent `dnaNucleotides` to be `undefined` :wink: !